Model/Sales/MagentoCollection.php

```diff
@@ -115,7 +115,8 @@ public function __construct(ObjectManagerInterface $objectManager, $request = nu
         $this->helper             = $objectManager->create(self::PATH_HELPER_DATA);
         $this->modelTrack         = $objectManager->create(self::PATH_ORDER_TRACK);
         $this->messageManager     = $objectManager->create(self::PATH_MANAGER_INTERFACE);
-        $this->myParcelCollection = (new MyParcelCollection())->setUserAgents(['Magento2'=> $this->helper->getVersion()]);
+        $this->myParcelCollection = (new MyParcelCollection())->setUserAgent('Magento2 ' .
```

Comment:
Waarom de spatie? Waarom niet `setUserAgents`? De origin in de api is `Magento`, niet `Magento2`. Dit werkt overigens niet als er ooit iets qua `filter` of `map` gedaan wordt op de collection.